src/index.ts

```diff
@@ -1551,7 +1569,55 @@ class Spanner extends GrpcService {
           },
         })
       );
-      callback(null, requestFn);
+
+      // Wrap requestFn so as to inject the spanner request id into
+      // every returned error, so that users can have debugging continuity.
```

Comment:
nit: Simplify comment

```suggestion
      // Wrap requestFn to inject the spanner request id into every returned error.
```

Reply:
Done, thanks!